### ISSUE TITLE
cobalt gets rate limited by video.bsky.app

### ISSUE THREAD

**Issue by vanontom** [imputnet/cobalt#928]
### bug description

Cobalt seems to have developed severe difficulties downloading Bluesky videos. Usually requires many attempts to successfully download a video (many times 3-5 tries, but sometimes over 10). Rarely downloads upon first attempt ("couldn't find any media that i could download for you. are you sure you pasted the right link?"). Sometimes downloads blank video (0 bytes). Sometimes downloads partial video (ouch, check duration). Eventually downloads properly (if patient enough).

Issue has been apparent for at least a week, possibly getting worse (more attempts needed). Bluesky introduced video only fairly recently, and has grown by millions of users lately, so not sure if that's related. Seems to affect any and all videos. Tested Firefox (default and custom settings) and Edge (default settings) with similar results.

### reproduction steps

1. go to https://bsky.app
2. find a video (search "#video" hashtag), click the post, and copy url
3. go to https://cobalt.tools, and paste url
4. wait for download. may receive pop-up ("couldn't find..."), or empty video, or partial video.
5. retry, again and again. eventually full video will download.

### links

```shell
https://bsky.app/profile/texasbirder.bsky.social/post/3laxinn6dx22h
https://bsky.app/profile/liamquane.bsky.social/post/3lawig3kyyc2y
https://bsky.app/profile/roppityphotos.bsky.social/post/3lax73nddqd2i
https://bsky.app/profile/cleme-3d.bsky.social/post/3lawkpxnej22b
https://bsky.app/profile/criscal.bsky.social/post/3laxcbrfwws2x
```

### platform information

Firefox 132.0 (Windows 11 23H2)

**Comment by wukko** [comment]
fixed via #929

**Comment by vanontom** [comment]
Ah, it was rate limiting. Thank you @wukko for checking this out, and fixing it ultra quickly!
